cookbook/how_to_apply_patch.rst

```diff
@@ -0,0 +1,119 @@
+.. index::
+    single: Patch
+
+How to apply patch
+==================
+
+The patch file name supports the following naming convention: {package}-{version_to_apply}.patch.
+For example, platform-1.9.2.patch.
+
+**1** Create backups of your Database and Code.
+
+**2** Stop the cron tasks
+
+.. code-block:: bash
+
+    $ crontab -e -uwww-data
+
+Comment this line.
+
+.. code-block:: text
+
+     */1 * * * * /usr/bin/php /srv/prod/crm-application/app/console --env=prod oro:cron >> /dev/null
+
+Kill the related processes.
```

Comment:
This looks like risky step, what if there are more than one application on the server? I believe this should be limited to the process associated with job daemon.